test/remoting.integration.js

```diff
@@ -73,60 +73,30 @@ describe('remoting - integration', function() {
   });
 
   describe('Model shared classes', function() {
-    function formatReturns(m) {
-      var returns = m.returns;
-      if (!returns || returns.length === 0) {
-        return '';
-      }
-      var type = returns[0].type;
-      return type ? ':' + type : '';
-    }
-
-    function formatMethod(m) {
-      return [
-        m.name,
-        '(',
-        m.accepts.map(function(a) {
-          return a.arg + ':' + a.type;
-        }).join(','),
-        ')',
-        formatReturns(m),
-        ' ',
-        m.getHttpMethod(),
-        ' ',
-        m.getFullPath(),
-      ].join('');
-    }
-
-    function findClass(name) {
-      return app.handler('rest').adapter
-        .getClasses()
-        .filter(function(c) {
-          return c.name === name;
-        })[0];
-    }
-
-    it('has expected remote methods', function() {
+    it('has expected remote methods with default model.settings.replaceOnPUT' +
+      'set to true (3.x)',
+    function() {
       var storeClass = findClass('store');
-      var methods = storeClass.methods
-        .filter(function(m) {
-          return m.name.indexOf('__') === -1;
-        })
-        .map(function(m) {
-          return formatMethod(m);
-        });
+      var methods = getMethods(storeClass.methods);
```

Comment:
I read this line as "get methods from storeClass methods", which sounds a bit unusual to me. How about

```js
var methdos = getFormattedMethodsExcludingRelations(storeClass);
```